Tuxboard.Core.Tests/ServiceCollectionExtensionsTests.cs

```diff
@@ -0,0 +1,87 @@
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Tuxboard.Core.Configuration;
+using Tuxboard.Core.Data.Context;
+using Tuxboard.Core.Infrastructure;
+using Tuxboard.Core.Infrastructure.Interfaces;
+
+namespace Tuxboard.Core.Tests;
+
+public class ServiceCollectionExtensionsTests
+{
+    [Fact]
+    public void AddTuxboardDashboard_WithValidConfiguration_AddsServices()
+    {
+        // Arrange
+        const string jsonConfig = """
+        {
+            "TuxboardConfig": {
+                "ConnectionString": "Server=(localdb)\\mssqllocaldb;Database=Tuxboard;Trusted_Connection=True;MultipleActiveResultSets=true",
+                "MigrationAssembly": "Tuxboard.Core",
```

Comment:
First off, thanks for the ServiceCollectionTests. Definitely helpful and was something on my to-do list.

One quick question: Is a "MigrationsAssembly" configuration going to change that much where it's required in an appsettings file? Trying to think of a scenario where it would be necessary as an appsetting?

Reply:
Not really, but any suggestions for obtaining the assembly name?

Reply:
In each of the Tuxboard.Examples where I use migrations (for example, the latest [11-Default-Widgets](https://github.com/jdanylko/Tuxboard.Examples/blob/master/11-Default-Widgets/Program.cs#L22)), I pass in a MigrationsAssembly in the UseSqlServer() method with the connection string

In the case of the DbContext being used in the main app, you could do an Assembly.GetExecutingAssembly()? Personally, in each example I've presented so far regarding MS Identity, I now have two DbContexts (MS Identity _and_ TuxDbContext). I would feel better just identifying which one to use for Migrations (as mentioned in the [post](https://www.danylkoweb.com/Blog/creating-default-dashboards-using-roles-U8)).

Honestly (now that I've had some sleep), it wouldn't matter if it was in the appsettings or hard-coded in the Program.cs. I don't expect it to change much when performing migrations.

Reply:
I applied your suggestion.